src/vue/modules/issuances-explorer/index.spec.js

```diff
@@ -0,0 +1,156 @@
+  let wrapper
+
+  beforeEach(() => {
+    store = new Vuex.Store({
+      modules: { 'issuances-explorer': issuancesExplorerModule },
+    })
+
+    wrapper = shallowMount(IssuancesExplorerModule, {
+      store,
+      localVue,
+      propsData: props,
+    })
+  })
+
+  describe('created hook', () => {
+    it('initializes the API', () => {
+      const spy = sinon.spy(Api, 'initApi')
+
+      shallowMount(IssuancesExplorerModule, {
+        localVue,
+        store,
+        propsData: props,
+      })
+
+      expect(spy.withArgs(props.wallet, props.config).calledOnce).to.be.true
+
+      spy.restore()
+    })
+
+    it('sets the account ID', () => {
+      const spy = sinon.spy(IssuancesExplorerModule.methods, 'setAccountId')
+
+      shallowMount(IssuancesExplorerModule, {
+        localVue,
+        store,
+        propsData: props,
+      })
+
+      expect(spy.withArgs(props.wallet.accountId).calledOnce).to.be.true
+
+      spy.restore()
+    })
+
+    it('initializes first issuances page loader', () => {
+      const spy = sinon.spy(IssuancesExplorerModule.methods, 'initFirstPageLoader')
+
+      shallowMount(IssuancesExplorerModule, {
+        localVue,
+        store,
+        propsData: props,
+      })
+
+      expect(spy.calledOnce).to.be.true
+
+      spy.restore()
+    })
+  })
+
+  describe('watcher', () => {
+    describe('shouldUpdate', () => {
+      it('should initialize page loader if the passed value is true', () => {
+        const spy = sinon.stub(wrapper.vm, 'initFirstPageLoader').resolves()
+
+        wrapper.setProps({ shouldUpdate: true })
+
+        wrapper.vm.$nextTick(_ => {
+          expect(spy.calledOnce).to.be.true
+          spy.restore()
+        })
+      })
+
+      it('should not initialize page loader if the passed value is false', () => {
+        const spy = sinon.stub(wrapper.vm, 'initFirstPageLoader').resolves()
+
+        wrapper.setProps({ shouldUpdate: false })
+
+        wrapper.vm.$nextTick(_ => {
+          expect(spy.notCalled).to.be.true
+          spy.restore()
+        })
+      })
+    })
+  })
+
+  describe('method', () => {
+    describe('loadIssuancesFirstPage', () => {
+      it('loads the account issuances', async () => {
+        const spy = sinon.stub(wrapper.vm, 'loadIssuances').resolves({
+          data: [],
+        })
+
+        await wrapper.vm.loadIssuancesFirstPage()
+
+        expect(spy.calledOnce).to.be.true
+
+        spy.restore()
+      })
+
+      it('sets isLoaded property to true if loading was succeded', async () => {
+        sinon.stub(wrapper.vm, 'loadIssuances').resolves({
+          data: [],
+        })
+
+        await wrapper.vm.loadIssuancesFirstPage()
+
+        expect(wrapper.vm.isLoaded).to.be.true
+
+        wrapper.vm.loadIssuances.restore()
+      })
+
+      it('handles the error if loading was failed', async () => {
+        sinon.stub(wrapper.vm, 'loadIssuances').throws()
+        const spy = sinon.stub(ErrorHandler, 'processWithoutFeedback')
```

Comment:
@max-nes assumed that such format:

```
sinon.stub(ErrorHandler, 'processWithoutFeedback')
expect(ErrorHandler.processWithoutFeedback).to.have.been.called.once
```

 is more readable because of using `sinon-chai` and asserting the method itself (not the returned spy).

I probably will often miss such stuff during reviews, so keep in mind pls

Reply:
Updated everywhere in the module tests